### PR TITLE
Created the web terminal

### PR DESCRIPTION
**Pull request by janupasilva2005**

### CODE REVIEW

File: Web-terminal/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Web-Terminal
+
```
Comment (avincodes) [inline Web-terminal/README.md:2]: Please add a screenshot of the snippet in action.

**Review by avincodes — CHANGES_REQUESTED** [review]

File: Web-terminal/README.md (hunk 1 above)
Comment (janupasilva2005) [reply Web-terminal/README.md:2]: image

Comment (avincodes) [reply Web-terminal/README.md:2]: Yes please add the Screenshot to the Readme!

Comment (janupasilva2005) [reply Web-terminal/README.md:2]: ok wait

Comment (janupasilva2005) [reply Web-terminal/README.md:2]: Ok check now

Comment (avincodes) [reply Web-terminal/README.md:2]: Nope, add the screenshot to the readme!

Comment (avincodes) [reply Web-terminal/README.md:2]: Sorry, just noticed that you have added it!

**Review by avincodes — APPROVED** [review]

**Comment by avincodes** [comment]
Great Work! Thank you!

**Pull request merged**